libs/datasets/timeseries.py

```diff
@@ -398,6 +399,20 @@ def timeseries(self) -> pd.DataFrame:
                 dtype="float",
             )
 
+    def location_ids(self) -> pd.Index:
+        return self.static.index.unique(CommonFields.LOCATION_ID).union(
+            self.timeseries_bucketed.index.unique(CommonFields.LOCATION_ID)
+        )
+
+    @cached_property
+    def geo_data(self) -> pd.DataFrame:
```

Comment:
hmm, how do you feel about loading geo_data on init, or providing a helper static function that loads it and passes it to init